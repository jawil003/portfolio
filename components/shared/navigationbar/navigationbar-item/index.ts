export { NavigationBarItem } from "./navigationbar-item";
export type { Props as NavigationBarItemProps } from "./navigationbar-item";
