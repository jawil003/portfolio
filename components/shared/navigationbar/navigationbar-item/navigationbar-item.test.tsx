import { render } from "@testing-library/react";
import { NavigationBarItem } from "./navigationbar-item";

describe("NavigationBarItem", () => {
  test("Create NavigationBarItem", () => {
    const testNavigationBarItem = render(
      <NavigationBarItem href="https://google.de" />,
    );

    expect(
      testNavigationBarItem.container
        .firstChild,
    ).toMatchSnapshot();
  });
});
