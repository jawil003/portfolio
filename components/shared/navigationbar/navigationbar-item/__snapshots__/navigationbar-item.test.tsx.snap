// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationBarItem Create NavigationBarItem 1`] = `
.emotion-0 {
  padding: 10px 20px;
  height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #000;
  cursor: pointer;
}

.emotion-0>svg {
  fill: #000;
}

.emotion-0:hover {
  background-color: rgba(
                  0,
                  0,
                  0,
                  0.35
                );
}

.emotion-0>* {
  display: inline-block;
}

.emotion-1 {
  text-align: start;
  font-style: normal;
  margin: unset;
  -webkit-text-decoration: none;
  text-decoration: none;
  letter-spacing: normal;
  word-spacing: normal;
  color: #000;
  word-wrap: unset;
  padding: unset;
  word-break: normal;
}

.emotion-1 {
  font-weight: 600;
  font-size: 1em;
}

<body>
  <div>
    <a
      class="emotion-0"
    >
      <span
        class="emotion-1 emotion-2"
        color="#000"
        id="subtitle1"
      />
    </a>
  </div>
</body>
`;
